docs/requirements.yaml

```diff
@@ -1,6 +1,6 @@
 name: propkatrajdocs
 channels:
-  - default
```

Comment:
As far as I can tell, *default* should have been *defaults* but because the docs.yaml specified `channels: conda-forge`, we always used conda-forge anyway.